package.json

```diff
@@ -27,7 +27,7 @@
     "clean:docs": "rimraf dist",
     "clean:preview": "nx clean storybook",
     "cleaner": "nx run-many --target clean --projects",
-    "compare": "nx run-many --verbose --target compare --projects",
+    "compare": "node ./tasks/compare-compiled-output.js",
```

Comment:
The script itself allows for better reporting on multiple component sets - the largest use-case of the script is verifying changes on the entire repo so better to use it directly than running it individually on each package (see the results when run `node ./tasks/compare-compiled-output.js` vs. `nx run-many --verbose --target compare --projects`